Documentation/sources/push-notification/how-to-display-iam-on-push-click.md

```diff
@@ -0,0 +1,30 @@
+# Display an in-message when a push notification is clicked
+
+## Overview
+This document describes the steps that allow a designated in-app message to be shown to an end user when they open the app through a push notification click.
+
+## Add a trigger in the In-App Message Campaign
```

Comment:
nit: be consistent with capitalization in our headings.  looks like we mostly don't need to capitalize any word other than the first for these sections since there are no proper nouns.